python/test/t_KissFFT_ND_std.py

```
@@ -0,0 +1,55 @@
+
+# Complex collection input (should match Point input)
+data2 = ot.ComplexCollection(vals)
+r4 = fft.transform(data2, [2, 3, 4])
+diff = max(abs(r4[i] - r1[i]) for i in range(len(vals)))
+assert diff < 1e-14, "Complex input mismatch"
+print("Complex input OK")
+
+# Check error on bad size
+try:
+    fft.transform(data, [2, 3, 5])
+    assert False, "Should have raised"
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)